eng/pipelines/templates/jobs/archetype-go-release.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  DependsOn: Build
+  ServiceDirectory: ''
+
+stages:
+  - ${{if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
```

Comment:
I think this conditional is unnecessary since you're also including it in the template include for the archetype-sdk-client.yml file.